src/utils/misc.ts

```diff
@@ -32,17 +32,23 @@ export const twirlLoader = () => {
 	}
 }
 
-export const parseUrl = (args: Record<string, unknown>): string => {
-	if (typeof args.url === 'string') {
-		if (args.url.includes('://')) {
-			return args.url
+export const parseRunUrl = (args: Record<string, unknown>) => {
+	if (typeof args.runUrl === 'string') {
+		let runUrl = args.runUrl
+		if (!runUrl.includes('://')) {
+			runUrl = `https://${runUrl}`
+		}
+
+		const matches = runUrl.match(/^(\S+)\/project\/(\w+)\/run\/(\d+)\/?$/)
```

Comment:
user can accidentally paste longer URL, that points to a specific test case in a test run: https://qas.eu1.qasphere.com/project/TEST/run/1/tcase/1 
(this is actually example from the documentation)

the difference is subtle, so I suggest we ignore everything else after run id.